### PR TITLE
Replace git misspell service with rojopolis spellchecker

### PR DESCRIPTION
**Pull request by genie9**
### Description

Replace git misspell service with rojopolis spellchecker https://github.com/rojopolis/spellcheck-github-actions.
This adds configuration file for spellchecking and wordlist file for words to exclude from checking. 

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### Changes Made

Github actions changed the use of https://git.io/misspell to rojopolis/spellcheck-github-actions.
- adds spellchecker config file  .spellcheck.yml
- adds .wordlist.txt with words to ignore from checking

Cleaned typos from the documentation.

### Testing

- [x] Tests do not apply

### CODE REVIEW

**Review by blankdots — APPROVED** [review]
super

File: .wordlist.txt [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+addObjectToDrafts
+addObjectToFolder
+Ajv
+api
+apisauce
+asciicast
+asciinema
+Async
+async
+autenticate
```
Comment (teemukataja) [inline .wordlist.txt:12]:
```suggestion
authenticate
```

File: .wordlist.txt [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+githubusercontent
+GroupedBySchema
+gunicorn
+hostname
+href
+html
+http
+https
+INITDB
+inteded
```
Comment (teemukataja) [inline .wordlist.txt:66]:
```suggestion
intended
```

File: .wordlist.txt [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+Keyfile
+Kubernetes
+li
+localhost
+lt
+makeStyles
+maxdepth
+md
+metadataObjects
+Metdata
```
Comment (teemukataja) [inline .wordlist.txt:83]:
```suggestion
Metadata
```

File: .wordlist.txt [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+middlewares
+modindex
+mongo
+Mongodb
+mongodb
+mTFEFsWsNUbP
```
Comment (teemukataja) [inline .wordlist.txt:89]: What is this? :D

File: .wordlist.txt [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+ObjectStatus
+ObjectSubmissionTypes
+ObjectTags
+objectType
+ObjectTypes
+OIDC
+ol
+OpenID
+openid
+overriden
```
Comment (teemukataja) [inline .wordlist.txt:109]:
```suggestion
overridden
```

File: .wordlist.txt [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+readthedocs
+redux
+REFERER
+reqs
+rootReducer
+rst
+schemas
+SDA
+sda
+seprately
```
Comment (teemukataja) [inline .wordlist.txt:127]:
```suggestion
separately
```

File: .wordlist.txt [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+SLLVERSION
+spesified
```
Comment (teemukataja) [inline .wordlist.txt:129]:
```suggestion
specified
```

**Review by teemukataja — CHANGES_REQUESTED** [review]

File: .wordlist.txt (hunk 4 above)
Comment (blankdots) [reply .wordlist.txt:89]: looks to be something in `validator.rst`

File: .wordlist.txt (hunk 7 above)
Comment (blankdots) [reply .wordlist.txt:129]: u don't see this anywhere in the code, is this needed ?

**Review by blankdots — CHANGES_REQUESTED** [review]
seems i missed some mistakes, @teemukataja caught them - and pretty sure i introduced most of them :disappointed: 
let us correct the spelling instead of adding exceptions to the dictionary

File: .wordlist.txt (hunk 4 above)
Comment (teemukataja) [reply .wordlist.txt:89]: Oh yea, it's part of an asciinema video id
```
  <script id="asciicast-FWYs48FhJ1mTFEFsWsNUbP43g" src="https://asciinema.org/a/FWYs48FhJ1mTFEFsWsNUbP43g.js" async></script>
```

File: .wordlist.txt (hunk 7 above)
Comment (genie9) [reply .wordlist.txt:125]: took away irrelevant string from wordlist

**Review by blankdots — APPROVED** [review]

**Pull request merged**